File: app/src/main/java/com/drkein/stayon/tools/L.java
package com.drkein.stayon.tools;

import android.util.Log;

import com.drkein.stayon.BuildConfig;

/**
 * Created by kein on 2017. 1. 14..
 */

public class L {
    public static void d(String TAG, String msg) {
        if (BuildConfig.DEBUG) Log.d(TAG, msg);
    }
}

File: app/src/main/java/com/drkein/stayon/tools/L.kt
package com.drkein.stayon.tools

import android.util.Log
import com.drkein.stayon.BuildConfig

/**
 * Created by kein on 2017. 1. 14..
 */

object L {

    @JvmStatic
    fun d(TAG: String, msg: String) {
        if (BuildConfig.DEBUG) Log.d(TAG, msg)
    }

}
